src/app/features/search/components/pages/index/IndexPage.tsx

```diff
@@ -1,17 +1,37 @@
-import React from "react"
+import React, { useState, useCallback } from "react"
 import { RouteComponentProps } from "@reach/router"
+import { SearchBar } from "@datapunt/asc-ui"
+import _ from "lodash"
 
 import Heading from "app/features/shared/components/atoms/Heading/Heading"
 import DefaultLayout from "app/features/shared/components/layouts/DefaultLayout/DefaultLayout"
-
 import SearchResults from "app/features/search/components/organisms/SearchResults/SearchResults"
 
-const IndexPage: React.FC<RouteComponentProps> = () => (
-  <DefaultLayout>
-    // TODO: Add SearchForm, remove Heading
-    <Heading>Zoeken</Heading>
-    <SearchResults />
-  </DefaultLayout>
-)
+const IndexPage: React.FC<RouteComponentProps> = () => {
+  const [searchString, setSearchString] = useState("")
+  const delayedQuery = useCallback(_.debounce(( q: any ) => sendQuery(q), 750), [])
```

Comment:
I would change this to:
`const delayedQuery = useCallback(_.debounce(setSearchString), 750), [ setSearchString ])`

Reply:
Done